### PR TITLE
Tw 2026 update settings UI

### PR DESCRIPTION
**Pull request by KhaledNjim**
## Ticket
#2026

## Impact description
Settings screen UI updated

## Resolved
**Attach screenshots or videos demonstrating the changes**
- Web:
2005_web
- Android:
mobile_2005

-
- IOS:

### CODE REVIEW

**Comment by hoangdat** [comment]
pleaase rebase, your branch is outdated

File: lib/pages/settings_dashboard/settings/settings_view_style.dart [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
-  static const EdgeInsetsDirectional titlePadding =
-      EdgeInsetsDirectional.only(start: 16);
-
+  static const EdgeInsetsDirectional titlePaddingMobile =
+      EdgeInsetsDirectional.only(top: 12, bottom: 11);
+  static const EdgeInsetsDirectional titlePaddingWeb =
+      EdgeInsetsDirectional.symmetric(horizontal: 16, vertical: 8);
   static EdgeInsetsDirectional itemBuilderPadding =
-      const EdgeInsetsDirectional.all(16.0);
+      const EdgeInsetsDirectional.only(top: 16.0, bottom: 16.0, start: 8.0);
```
Comment (hoangdat) [inline lib/pages/settings_dashboard/settings/settings_view_style.dart:18]: should 8 for all

Comment (KhaledNjim) [reply lib/pages/settings_dashboard/settings/settings_view_style.dart:18]: It has 16 at the top and bottom 
image

**Comment by KhaledNjim** [comment]
> pleaase rebase, your branch is outdated

done

**Comment by hoangdat** [comment]
LGTM

**Review by nqhhdev — APPROVED** [review]
LGTM

**Review by hoangdat — APPROVED** [review]

**Pull request merged**